software/binpastes.yml

```
@@ -0,0 +1,13 @@
+name: BinPastes
+website_url: https://github.com/querwurzel/BinPastes
+description: Minimal pastebin also supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment.
+licenses:
+  - Apache-2.0
+platforms:
+  - Java
+tags:
+  - Pastebins
+source_code_url: https://github.com/querwurzel/BinPastes
+stargazers_count: 2
```

These will be filled in automatically, so feel free to remove them. 
```suggestion
- stargazers_count: 2
- updated_at: '2023-11-13'
- archived: false
```